### DISCUSSION TITLE
[Saran] Format Timestamp Response dalam GMT+7 / WIB

### DISCUSSION THREAD

**Discussion by microsauruz** [kirimwa-id/forum#105] [Saran dan Ide]
Hallo mas Rio,
boleh usul saran,
berhubung format timestamp pada response luaran API nya saya lihat masih belum GMT+7, bila memungkinkan, kali2 bisa diubah ke GMT+7 saja mas, biar lebih enak kalau dilihat dan enggak ngebikin bingung.
Soalnya sesekali saya mantau dari HP melihat apakah HP saya masih tersambung atau sedang putus.

![Screenshoot](https://user-images.githubusercontent.com/29596042/156726211-2b8d8ddb-740a-47ff-b6ea-e7c752a99c3d.JPG)

**Comment by rioastamal** [comment]
@microsauruz Terima kasih - akan dipertimbangkan, namun tidak bisa janji rilis kapan :)

**Comment by shahib-mirbath** [comment]
> Hallo mas Rio,
> boleh usul saran,
> berhubung format timestamp pada response luaran API nya saya lihat masih belum GMT+7, bila memungkinkan, kali2 bisa diubah ke GMT+7 saja mas, biar lebih enak kalau dilihat dan enggak ngebikin bingung.
> Soalnya sesekali saya mantau dari HP melihat apakah HP saya masih tersambung atau sedang putus.
> 
> ![Screenshoot](https://user-images.githubusercontent.com/29596042/156726211-2b8d8ddb-740a-47ff-b6ea-e7c752a99c3d.JPG)
> 
> 

Di saya, saya ubah saat ditampilkan, sesuai waktu lokasi yg kita sett.
